source/blog/2017-07-21-rpm-ostree-v2017.7-released.html.md

```diff
@@ -0,0 +1,273 @@
+---
+title: rpm-ostree v2017.7 released
+author: jlebon
+date: 2017-07-24 14:00:00 UTC
+tags: rpm-ostree, rpm, ostree, atomic, atomic host
+comments: true
+published: true
+---
+
+[rpm-ostree](https://rpm-ostree.readthedocs.org/) is the
+hybrid image/package system that provides transactional
+upgrades on Atomic Host. Here are some highlights from
+version
+[v2017.7](https://github.com/projectatomic/rpm-ostree/releases/tag/v2017.7),
+including PolicyKit and experimental overrides support.
+
+READMORE
+
+As usual, you can test out this release by
+[rebasing your Fedora Atomic Host](https://fedoraproject.org/wiki/QA:Updates_Testing#Using_it_with_Atomic_Host_.28Fedora_24_and_later.29)
+onto the testing branch. Feel free to also leave karma in
+the pending Bodhi updates:
+
+- [Fedora 25 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-45adcbb9a3)
+- [Fedora 26 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-483c7979ee)
+
+Note that this is the final release on Fedora 25.
+
+### Experimental support for overrides
+
+The most visible change in this release is the addition of a
+new `ex` command: `rpm-ostree ex override`. As with all
+experimental additions, please note that the interface
+described below is subject to change in future versions.
+
+Whereas the `install` command allows you to overlay packages
+on top of the base commit received from your content
+provider, the `override` command allows you to modify the
+set of packages in the base commit itself. For example, you
+may want to remove a package that conflicts with an overlay,
+or more interestingly, update a package to a newer version
+to fix a bug.
+
+Of course, such modifications should be done with care,
+since they in effect "void" the warranty implied by an
+ostree commit. However, these powerful semantics enable us
+to take the image/package hybrid paradigm to a new level.
+Thanks to rpm-ostree, you can easily keep track of every
+deviation from the base image.
+
+The `override` command supports two subcommands: `remove`
+and `replace`.
+
+### Removal overrides
+
+Not surprisingly, the `remove` subcommand allows you remove
+a base package. Let's see it in action!
+
+The host below has `strace` installed and no packages
+layered:
+
+```
+# rpm-ostree status
+State: idle
+Deployments:
+* vmcheck
```

Comment:
Let's do demos with the main fedora refspec?   You could probably just replace this with the current prod output of `rpm-ostree status` since few people are going to understand the `vmcheck`.